cmake/PostgreSQLExtension.cmake

```diff
@@ -171,6 +183,8 @@ function(add_postgresql_extension NAME)
                 add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../${dependency}" "${CMAKE_CURRENT_BINARY_DIR}/${dependency}")
             elseif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../../${dependency}")
                 add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../${dependency}" "${CMAKE_CURRENT_BINARY_DIR}/${dependency}")
+            elseif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../../misc/${dependency}")
```

Comment:
Do we still need this with `ninja` not being an explicit dependency anymore?

Reply:
No, we don't need. Dropped that `elseif`.